### ISSUE TITLE
overflow:hidden is not working within .media object

### ISSUE THREAD

**Issue by ismailfarooq** [twbs/bootstrap#19719]
css [truncate](https://css-tricks.com/snippets/css/truncate-string-with-ellipsis/) not working in media if i remove media-body class so truncate  works but not media. You can see here https://jsfiddle.net/sqmweeLn/

**Comment by twbs-lmvtfy** [comment]
Hi @ismailfarooq!

You appear to have posted a live example (http://s.bootply.com/render/SNnkpXZZDU), which is always a good first step. However, according to [the HTML5 validator](http://html5.validator.nu), **your example has some validation errors**, which might potentially be causing your issue:
* line 91, column 5 thru column 11: End tag for  ```` body ```` seen, but there were unclosed elements.
* line 56, column 1 thru column 5: Unclosed element ```` div ````.
* line 43, column 9 thru column 30: Unclosed element ```` div ````.

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by cvrebert** [comment]
Approximate duplicate of #15622.

**Comment by cvrebert** [comment]
Actually, make that duplicate of #15958.
